### PR TITLE
Document 'healthcheck' public endpoint

### PR DESCRIPTION
**Pull request by ozkriff**
See https://github.com/exonum/exonum-testkit/pull/20

### CODE REVIEW

File: src/advanced/node-management.md [hunk 1]
```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
...
+
+#### Parameters
+
+None.
+
+#### Response
+
+JSON object with the following fields:
+
+- **connectivity**: bool
```
Comment (slowli) [inline src/advanced/node-management.md]: You need to add two spaces after `bool` in order to make a line-break, similar to how it's done in other endpoint descriptions.

**Comment by ozkriff** [comment]
Fixed

File: src/advanced/node-management.md [hunk 2]
```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
 }
 ```
 
+### Healthcheck
+
+```none
+GET {system_base_path}/healthcheck
+```
+
+Returns a boolean representing if the node is connected to other peers.
```
Comment (Maria-Nosyk) [inline src/advanced/node-management.md]: I'd rather say "...a boolean value representing..."

File: src/advanced/node-management.md [hunk 3]
```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
...
+
+#### Parameters
+
+None.
+
+#### Response
+
+JSON object with the following fields:
```
Comment (Maria-Nosyk) [inline src/advanced/node-management.md]: > JSON object --> A JSON object

File: src/advanced/node-management.md [hunk 4]
```diff
@@ -168,6 +168,33 @@ JSON object with the following fields:
...
+
+- **connectivity**: bool  
+  Is the node connected to other peers?
```
Comment (Maria-Nosyk) [inline src/advanced/node-management.md]: I'd rather paraphrase it as a statement "Indicates whether the node is connected to the other peers."

**Comment by ozkriff** [comment]
Thanks, fixed.

**Pull request merged**